client/components/jetpack-header/style.scss

```diff
@@ -6,3 +6,10 @@
 .jetpack-header > svg {
 	max-width: 100%;
 }
+
+.is-section-jetpack-connect,
+.is-jetpack-login {
+	.jetpack-header__partner-logo-plus .gridicon {
+		color: #bbc9d5;
+	}
+}
```

Comment:
Nitpick: This file should end with an empty newline.

Reply:
Thanks, added an empty line in https://github.com/Automattic/wp-calypso/pull/31989/commits/cd645c3b9e54f1da7e96cb6a43e4f78ba6b5951d